packages/app/screens/group/tabs/MembersScreen.js

```diff
@@ -1,8 +1,51 @@
+import MemberCard from "@app/components/MemberCard/MemberCard";
 import React from "react";
 import { View } from "react-native";
 
+
+// max characters display should be 15?
+const mockData = [
+  {
+    key: "one",
+    name: "Carl",
+  },
+  {
+    key: "two",
+    name: "Joe",
+  },
+  {
+    key: "three",
+    name: "Bobby Jones 3rd",
+  },
+  {
+    key: "four",
+    name: "Chicken Nugget",
+  },
+  {
+    key: "five",
+    name: "Jill",
+  },
+  {
+    key: "six",
+    name: "Jack",
+  },
+  {
+    key: "seven",
+    name: "Bartholemew"
+  }
+];
+
 function MembersScreen() {
-  return <View></View>;
+
+  return (
+    <View>
+      {mockData.map((section) => (
+        <View key={section.key}>
+          <MemberCard name={section.name.length <= 12 ? section.name : section.name.substring(0, 12) + "..."} />
```

Comment:
Really great solution for now! Just a comment, but we'll have to figure out another way to display user names in the future. Our UI/UX members may be able to come up with a better solution or we may need to limit username length.
https://ux.stackexchange.com/questions/37616/should-a-username-be-truncated/